sdk/src/client/secret/ledger_nano.rs

```diff
@@ -407,8 +407,9 @@ impl SecretManage for LedgerSecretManager {
     async fn sign_transaction(
         &self,
         prepared_transaction_data: PreparedTransactionData,
+        slot_index: impl Into<SlotIndex> + Send,
```

Comment:
Do we actually need this as an extra parameter? Expiration and Timelock require a Commitment Input in the transaction, and then we can just get the slot index from there, no?

Reply:
Done